test/compilable/cppmangle.d

```diff
@@ -1275,3 +1275,22 @@ version (Win64) extern(C++)
     extern(C++, struct) class DefaultClass20700_2 {}
     static assert(test20700_4.mangleof == `?test20700_4@@YAXPEAU?$TStruct20700_2@PEAUDefaultClass20700_2@@VDefaultStruct20700_2@@@@@Z`);
 }
+
+/*****************************************/
+
+alias noreturn = typeof(*null);
+
+extern (C++)
+{
+    alias fpcpp = noreturn function();
+    int funccpp(fpcpp);
+
+    version (Posix)
+        static assert(funccpp.mangleof == "PosixFunc";
```

Comment:
This won't pass.

Reply:
That type of comment isn't very helpful as it doesn't explain why.

Reply:
It doesn't need explaining because it is deliberately wrong for debugging purposes.

Reply:
done